platform/telemetry/src/TelemetrySubscription.js

```diff
@@ -239,6 +239,10 @@ define(
 
             initialize();
             this.unlistenToMutation = addMutationListener();
+
+            // Expose makeDatum for TelemetryHandler to use, but not
+            // as part of public API
+            this.makeDatum = makeDatum;
```

Comment:
Hrm, I don't think a method being on the prototype is necessarily the determinant of whether it's public or not-- would prefer to move all methods to prototype and tag them as `@private` for jsdoc (or adopt something like `_` prefix for `"private"`).